index.js

```diff
@@ -482,6 +482,120 @@ app.post('/api/heartbeat', Validator.body(['coreid', 'event', 'data']).exists(),
   }
 })
 
+app.post('/api/siren-addressed', Validator.body(['coreid', 'event', 'data']).exists(), async (request, response) => {
```

Comment:
I'd prefer camelCase for the route name, so `/api/siredAddressed` to match the style of `/api/sensorEvent`.